docs/overview/architecture.md

```diff
@@ -113,4 +113,28 @@ When the user searches a vector from Vald:
 
 <!-- ### Update -->
 
-<!-- ### Delete -->
+<!-- ### Upsert -->
+
+### Delete
+
+<img src="../../assets/docs/delete_flow.png" />
+
+When the user delete a vector which is indexed in Vald Agent:
+
+1. Vald Ingress receives the delete request from the user. The request includes the vector ID(s) which is specified by the user.
+2. Vald Ingress will forward the request to the Vald Filter Gateway.
+3. Vald Filter Gateway will forward the request to the Vald Meta Gateway. Vald Meta Gateway is used to resolve the internal used UUID from Vald Meta to the user inserted vector ID in the Insert Step.
+4. Vald Meta Gateway will forward the request to the Vald Meta to confirm whether metadata which contains request ID(s) is existing or not.
+5. Vald Meta get the UUID(s) from the request ID(s). When the UUID(s) be not found, return with the error.
+6. If Vald Meta Gateway gets the UUID(s), Vald Meta Gateway will forward to the request with the UUID(s) to the Vald LB Gateway through the Vald Backup Gateway.
```

Comment:
I think number 6 is not a connection to LB Gateway, but a connection to Backup Gateway:thinking:

Reply:
Confirm Location information with Backup Gateway and send Remove request to LB Gateway. Then, when the request is successful, the Backup data is deleted.

Reply:
Now, you're right that Vald Backup Gateway will get Agent IP using UUID.
https://github.com/vdaas/vald/blob/master/pkg/gateway/vald/handler/grpc/handler.go#L741

But, in this article, we're targeting the new architecture, like #579 .(I cannot source code...)
And, the author, @kpango told us that getting location info from Vald Backup Gateway is too cost so he decide broadcast all agent with UUID instead of that.